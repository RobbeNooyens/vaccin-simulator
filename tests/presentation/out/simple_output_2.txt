Hub (143000): 
  -> Center 1 (4679 vaccins)
  -> Center 2 (0 vaccins)
  -> Center 3 (0 vaccins)

Center 1: 130521 gevaccineerd, nog 0 inwoners niet gevaccineerd
Center 2: 656920 gevaccineerd, nog 866120 inwoners niet gevaccineerd
Center 3: 344940 gevaccineerd, nog 405200 inwoners niet gevaccineerd
