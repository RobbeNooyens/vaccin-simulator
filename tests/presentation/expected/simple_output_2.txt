Hub (143000): 
  -> Center 1 (1100 vaccins)
  -> Center 2 (989 vaccins)
  -> Center 3 (0 vaccins)

Center 1: 5043 gevaccineerd, nog 0 inwoners niet gevaccineerd
Center 2: 8011 gevaccineerd, nog 0 inwoners niet gevaccineerd
Center 3: 0 gevaccineerd, nog 3021 inwoners niet gevaccineerd
